python/xorbits/_mars/dataframe/sort/sort_index.py

```diff
@@ -225,7 +225,13 @@ def sort_index(
         raise TypeError(f"Invalid na_position: {na_position}")
     psrs_kinds = _validate_sort_psrs_kinds(psrs_kinds)
     axis = validate_axis(axis, a)
-    level = level if level is None or isinstance(level, (list, tuple)) else [level]
+    level = (
```

Comment:
This is a little confusing. Better rewrite with the normal if-else syntax.

Reply:
done, thx